mne/io/edf/edf.py

```diff
@@ -653,31 +666,28 @@ def _read_edf_header(fname, exclude):
         channels = list(range(nchan))
 
         # read in 16 byte labels and strip any extra spaces at the end
-        ch_labels = [fid.read(16).strip().decode('latin-1')
-                     for ch in channels]
+        ch_labels = [fid.read(16).strip().decode('latin-1') for _ in channels]
 
         # get channel names and optionally channel type
         # EDF specification contains 16 bytes that encode channel names,
         # optionally prefixed by a string representing channel type separated
         # by a space
-        ch_names = []
-        ch_types = []
-        for ch_idx, this_label in enumerate(ch_labels):
-            # if no channel type, then we will default to eeg
-            ch_type = None
-            ch_name = this_label
-
-            # space is found, so the prefix is the channel type. 'Annotations'
-            # is also a keyword we search for
-            if (this_label.count(' ') == 1) and \
-                    ('Annotations' not in this_label):
-                ch_type, ch_name = this_label.split(' ')
-
-                # channel types should be upper case for easy comparison
-                ch_type = ch_type.upper()
-
-            ch_names.append(ch_name)
-            ch_types.append(ch_type)
+        if infer_types:
+            ch_types, ch_names = [], []
+            for ch_label in ch_labels:
+                ch_type, ch_name = 'EEG', ch_label  # default to EEG
+                parts = ch_label.split(' ')
```

Comment:
I'm not too familiar with the standard: is there any chance of channel names having tab characters?  If so then 
```suggestion
                parts = ch_label.split()
```
will split on any whitespace

Reply:
It's only spaces: https://www.edfplus.info/specs/edftexts.html#label